src/com/magento/idea/magento2plugin/completion/xml/XmlCompletionContributor.java

```diff
@@ -71,6 +71,13 @@ public XmlCompletionContributor() {
                 new PhpClassCompletionProvider()
         );
 
+        // <frontend_model>completion</frontend_model>
+        extend(CompletionType.BASIC,
+                psiElement(XmlTokenType.XML_DATA_CHARACTERS)
+                        .inside(XmlPatterns.xmlTag().withName(SystemXml.XML_TAG_FRONTEND_MODEL)),
+                new PhpClassCompletionProvider()
+        );
```

Comment:
Hi @konarshankar07 
We need completion for both - the attribute and the tag and not only in the system.xml. See the example: https://github.com/magento/magento2-phpstorm-plugin/pull/95/files 
Thank you!

Reply:
Hello @VitaliyBoyko ...
I checked for `frontend_model` XML attribute in magento codebase and we are not using the `frontend_model` attribute.
Thanks